.github/ISSUE_TEMPLATE/feature_request_ux.md

```diff
@@ -1,6 +1,6 @@
 ---
-name: Feature request for GitHub Copilot Chat
-about: Suggest an idea for Copilot Chat's interface in VS Code
+name: Feature request for GitHub Copilot Chat or completions

```

Comment:
I was envisioning separate templates for completions feature requests or issues/bug reports. Certain folks could then be specifically assigned to all completions requests and issues.

Reply:
Got it. Will create a new template for completions!